docker-compose.dev.yml

```diff
@@ -12,13 +12,3 @@ services:
       context: ./core/gf
       cache_from:
         - acctext/gf:latest
```

Comment:
Yes, because there's docker-compose.front-end.yml which can be simply combined to produce both front-end and backend. 

Having frontend inside docker-compose.dev.yml makes no way of running frontend separate (while actually developing it) from backend